blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("blog/", include("blog.urls", namespace="blog")),
+    path("", include("django.contrib.auth.urls")),
+] + debug_toolbar_urls()
```

Comment:
Here, you are trying to call 'debug_toolbar_urls' as a function, but it's not a function, it's a module. You should include it in your urlpatterns like this: 'path('__debug__/', include(debug_toolbar_urls))'.